packages/slice-machine/components/AppLayout/Navigation/index.tsx

```diff
@@ -13,6 +14,7 @@ export interface LinkProps {
   Icon: IconType;
   delimiter?: boolean;
   target?: "_blank";
+  RightElement?: React.ReactNode;
```

Comment:
OPT: right element doesn't seem very intuitive, i think a name that represent the visual element would be easier to understand for other devs.

Reply:
How about `RightIndicator`? I wanted to make it more generic, so that its not exclusive to the changes icon